common/src/main/java/io/pravega/schemaregistry/common/NameUtil.java

```diff
@@ -59,4 +59,17 @@ public static String qualifiedName(String qualifier, String name) {
         Preconditions.checkNotNull(name, "Name cannot be null");
         return Strings.isNullOrEmpty(qualifier) ? name : String.format("%s.%s", qualifier, name);
     }
+
+    /**
+     * Type value if the 'type' is not null of empty.
+     * If type is null or empty then the created name is simply 'namespace.groupName'.
+     *
+     * @param type the value provided with API call (schemaInfo.getType()).
+     * @param groupName the name of the group for schema
+     * @param namespace the namespace for the schema
+     * @return Provided name or Created name for type in SchemaInfo
+     */
+    public static String createTypeIfAbsent(String type, String groupName, String namespace) {
```

Comment:
I don;t see the need for this new method.

Reply:
This removes the redundancy in the earlier code. Further, if need be this can be utilized in future too

Reply:
IMO, this method is unnecessary. We could do with directly using `NameUtil.qualifiedName(...);`